packages/pages-shared/metadata-generator/parseRedirects.ts

```diff
@@ -119,6 +119,18 @@ export function parseRedirects(input: string): ParsedRedirects {
 		}
 		seen_paths.add(from);
 
+		if (status === 200) {
+			// Error can only be that it's not relative - given validateUrl is called above without onlyRelative:true,
+			// so if it's present, we can error to the user that proxying only expects relative urls
+			const [_, error] = validateUrl(to, true, true, true);
+			if (error)
+				invalid.push({
+					line,
+					lineNumber: i + 1,
+					message: `Proxy (200) redirects can only point to relative paths. Got ${to}`,
+				});
+		}
+
 		rules.push({ from, to, status, lineNumber: i + 1 });
```

Comment:
```suggestion
			const [proxyTo, error] = validateUrl(to, true, true, true);
			if (error) {
				invalid.push({
					line,
					lineNumber: i + 1,
					message: `Proxy (200) redirects can only point to relative paths. Got ${to}`,
				});
				continue;
			}
		}

		rules.push({ from, to: proxyTo, status, lineNumber: i + 1 });
```

Quite a sneaky `continue` that you need there with invalid rules 🥷

Reply:
It would be good to put an example of this in `parseRedirects.valid.test.ts`. Likely would have caught this :)